version: '3'
services:
  sync:
    image: wazopbx/wait
    environment:
      TIMEOUT: "${INTEGRATION_TEST_TIMEOUT}"

  auth:
    image: "wazopbx/wazo-auth-mock"
    ports:
      - "9497"

  provd:
    image: wazopbx/xivo-provd
    volumes:
      - "../..:/usr/src/xivo-provisioning/"
      - "./etc/xivo-provd/config.yml:/etc/xivo-provd/config.yml"
      - "./provd/configs/base:/var/lib/xivo-provd/jsondb/configs/base"
      - "./provd/configs/default:/var/lib/xivo-provd/jsondb/configs/default"
      - "./provd/configs/defaultconfigdevice:/var/lib/xivo-provd/jsondb/configs/defaultconfigdevice"
      - "./provd/configs/autoprov:/var/lib/xivo-provd/jsondb/configs/autoprov"
    ports:
      - "8666"
      - "8667"
      - "69/udp"

  pluginserver:
    image: trinitronx/python-simplehttpserver
    volumes:
      - "./provd/plugins/pkgs:/var/www"
    ports:
      - "8080"
